docs/data/routes/docs/nextjs/graphql/component/en.md

```diff
@@ -0,0 +1,41 @@
+---
+name: component
+routeTemplate: ./data/component-templates/article.yml
+title: Using GraphQL in component SSR/SSG
+---
+# Walkthrough: Using Component-Level GraphQL in SSR/SSG
+
+The Next.js sample app provides support for [component-level data fetching](/docs/nextjs/data-fetching/component-level-data-fetching), which enables executing GraphQL queries at the component level.
+
+> NOTE: If you are following the `Code First` developer workflow, you can execute GeaphQL component-level queries only in `Connected Mode`. They are not supported in [`Disconnected Mode`](/docs/techniques/working-disconnected/disconnected-overview).
+
+Let's go through the steps to execute first GraphQL query in your component. We can use [GraphQL-ConnectedDemo](https://github.com/Sitecore/jss/blob/master/samples/nextjs/src/components/graphql/GraphQL-ConnectedDemo.tsx) example.
+
+1. Define `getStaticProps` or/and `getServerSideProps`. Next steps you should perform inside these functions.
+2. Create new GraphQL client. `GraphQLClientFactory` is using [apollo-client](https://www.apollographql.com/docs/react) library.
```

Comment:
Should we provide a sentence about what Apollo is and why we're using this particular library? I see you link to Apollo docs later on in the Walkthrough - I would collect the Apollo background data into a paragraph before the walkthrough

Reply:
Added a little bit about Apollo: https://github.com/Sitecore/jss/pull/587/files#diff-f48c7acb9e727a477be3fca37355f49120f17b315fcb9c47421d6c37dff77bd6R12-R13
We also will have article: `Walkthrough: Using GraphQL on the client`. I expected that Apollo should be described there.